packages/server/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     // https://github.com/tsconfig/bases/blob/main/bases/node18.json
     "lib": ["es2023", "DOM", "DOM.Iterable"],
     "target": "es2022",
-    "module": "node16"
+    "module": "Node16",
+    "moduleResolution": "node16"
```

Comment:
we want Bundler i think? given that we transpile with rollup

Reply:
Not sure tbh - it just started complaining and I made it stop with this change